### DISCUSSION TITLE
P8 - Conflict between date and date_res?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1439] [Q&A] [answered]
I successfully ran the model for 24 hours and restarted to run another 24. However, in attempting to run for 48 hours then restart run for another 48 I hit the following error:

fcst_initialize ERROR: date /= date_res
                       date     = 2012   1   2   0   0   0
                       date_res = 2012   1   3   0   0   0

I'm sure there is something in the various configuration files that I missed that is set to 24 hours but I'm not sure what it is. What controls the internal model date that would be conflicting with date_res?

**Comment by benjamin-cash** [comment]
@ShawnCebulaNOAA - Could you help run down this answer?

**Reply by ShawnCebulaNOAA** [reply]
Sure thing!  I'll see what I can get for you Benjamin!

Shawn Cebula - EPIC End User Support

**Comment by benjamin-cash** [comment, marked as answer]
Marking this as answered - I tracked it down to a problem with fhrot.
